src/main/java/wolox/training/models/Book.java

```diff
@@ -6,6 +6,8 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
 import javax.persistence.SequenceGenerator;
 
 /**
```

Comment:
you should declare this class as table.
@Table(name = "books")

Reply:
I added this annotation